packages/apis/ipfs/src/mutation/schema.graphql

```diff
@@ -0,0 +1,11 @@
+#import { Mutation, ResponseType } into Http from "w3://ens/http.web3api.eth"
+
+type Mutation {
+  addFile(ipfsUrl: String!, fileName: String!, data: Bytes!): AddResult!
```

Comment:
Based on the IPFS HTTP API:
https://docs.ipfs.io/reference/http/api/#api-v0-add
https://github.com/ipfs-shipyard/js-ipfs-http-client-lite/blob/HEAD/API.md#add

I think we can add the following methods to have a more "complete" add implementation:
- Create an `AddFileOptions` type which includes some supported options (like `wrap-with-directory`). If `wrap-with-directory` is specified, the fileName argument is a valid argument and shouldn't be removed (as specified in my prior comment).
- Rename `AddResult` to `AddFileResult`.
- Create an `addFolder` method, which takes an array of folders & files as an argument instead of the fileName + data.